plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -659,6 +656,12 @@ open class HtmlRenderer(
 
     private fun resolveLink(link: String, page: PageNode): String = if (URI(link).isAbsolute) link else page.root(link)
 
+    private fun pathToRootPage(currentPage: PageNode): String =
+        context.configuration.homePage?.toASCIIString()
```

Comment:
Do we need `toASCIIString()` ? IMHO just `toString()` would be sufficient, I see no reason not to support all Unicode characters, but maybe I don't see something